src/App.tsx

```diff
@@ -1,10 +1,19 @@
 import React from "react";
-import Landing from "@landing";
+import { Landing } from "./pages";
+import { SignIn, About } from "@components/index";
+import { Routes, Route } from "react-router-dom";
 
 const App = () => {
   return (
     /*Private routing will be added once main screen is done*/
-    <Landing />
+    <>
+      <Routes>
+        <Route path="/" element={<Landing />}>
+          <Route path="sign-in" element={<SignIn />} />
```

Comment:
WDYT if you create a `constants` file where you save all your paths? I'll make your code easier to change in the future in cass you need it

Reply:
I like the idea, thanks!